ci-operator/config/openshift/microshift/openshift-microshift-main.yaml

```
@@ -111,73 +111,73 @@ tests:
   steps:
     cluster_profile: gcp
     workflow: openshift-microshift-e2e-rpm-install
 - as: e2e-openshift-conformance-sig-api-machinery
-  optional: true
+  optional: false
   run_if_changed: ^(cmd/.*|pkg/.*|packaging/.*|vendor/.*|go\.sum|go\.mod|Makefile|.*\.sh)
   steps:
     cluster_profile: gcp
     workflow: openshift-microshift-e2e-openshift-conformance-sig-api-machinery
 - as: e2e-openshift-conformance-sig-apps
-  optional: true
+  optional: false
   run_if_changed: ^(cmd/.*|pkg/.*|packaging/.*|vendor/.*|go\.sum|go\.mod|Makefile)
   steps:
     cluster_profile: gcp
     workflow: openshift-microshift-e2e-openshift-conformance-sig-apps
 - as: e2e-openshift-conformance-sig-arch
-  optional: true
+  optional: false
   run_if_changed: ^(cmd/.*|pkg/.*|packaging/.*|vendor/.*|go\.sum|go\.mod|Makefile)
   steps:
     cluster_profile: gcp
     workflow: openshift-microshift-e2e-openshift-conformance-sig-arch
 - as: e2e-openshift-conformance-sig-auth
-  optional: true
+  optional: false
   run_if_changed: ^(cmd/.*|pkg/.*|packaging/.*|vendor/.*|go\.sum|go\.mod)|Makefile
   steps:
     cluster_profile: gcp
     workflow: openshift-microshift-e2e-openshift-conformance-sig-auth
 - as: e2e-openshift-conformance-sig-cli
-  optional: true
+  optional: false
   run_if_changed: ^(cmd/.*|pkg/.*|packaging/.*|vendor/.*|go\.sum|go\.mod|Makefile)
   steps:
     cluster_profile: gcp
     workflow: openshift-microshift-e2e-openshift-conformance-sig-cli
 - as: e2e-openshift-conformance-sig-instrumentation
-  optional: true
+  optional: false
   run_if_changed: ^(cmd/.*|pkg/.*|packaging/.*|vendor/.*|go\.sum|go\.mod|Makefile)
   steps:
     cluster_profile: gcp
     workflow: openshift-microshift-e2e-openshift-conformance-sig-instrumentation
 - as: e2e-openshift-conformance-sig-network
-  optional: true
+  optional: false
   run_if_changed: ^(cmd/.*|pkg/.*|packaging/.*|vendor/.*|go\.sum|go\.mod|Makefile)
   steps:
     cluster_profile: gcp
     workflow: openshift-microshift-e2e-openshift-conformance-sig-network
 - as: e2e-openshift-conformance-sig-node
-  optional: true
+  optional: false
   run_if_changed: ^(cmd/.*|pkg/.*|packaging/.*|vendor/.*|go\.sum|go\.mod|Makefile)
   steps:
     cluster_profile: gcp
     workflow: openshift-microshift-e2e-openshift-conformance-sig-node
 - as: e2e-openshift-conformance-sig-scheduling
-  optional: true
+  optional: false
   run_if_changed: ^(cmd/.*|pkg/.*|packaging/.*|vendor/.*|go\.sum|go\.mod|Makefile)
   steps:
     cluster_profile: gcp
     workflow: openshift-microshift-e2e-openshift-conformance-sig-scheduling
 - as: e2e-openshift-conformance-sig-storage
-  optional: true
+  optional: false
   run_if_changed: ^(cmd/.*|pkg/.*|packaging/.*|vendor/.*|go\.sum|go\.mod|Makefile)
   steps:
     cluster_profile: gcp
     workflow: openshift-microshift-e2e-openshift-conformance-sig-storage
 - as: e2e-reboot
-  optional: true
+  optional: false
```

```suggestion
  optional: true
```

Let's make this one optional for now until we figure out why is it always failing